### PR TITLE
add AWS info to icgc-25k-data.md

### PR DESCRIPTION
**Pull request by edsu7**
- edited description to include AWS bucket
- noted which has controlled and public data
- AWS details and connection commands
- added ssh-rsa option to SFTP command

### CODE REVIEW

File: docs/data-access/icgc-25k-data.md [hunk 1]
```diff
@@ -22,15 +22,18 @@ Files from other contributing projects are all hosted by ICGC's partner reposito
 
 ## Accessing ICGC 25K Release Data
 
-A SFTP server is available to access ICGC Release Data and PCAWG data.
+ICGC Release Data and PCAWG data are hosted in two locations: a SFTP sever and an AWS bucket.
```
Comment (justincorrigible) [inline docs/data-access/icgc-25k-data.md:25]:
```suggestion
ICGC Release Data and PCAWG data are hosted in two locations: an SFTP server, and an AWS bucket.
```

File: docs/data-access/icgc-25k-data.md [hunk 2]
```diff
@@ -22,15 +22,18 @@ Files from other contributing projects are all hosted by ICGC's partner reposito
...
 
-The server hosts three data directories with the following data:
+The SFTP contains public and DACO controlled data, only available to authorized DACO-approved users only. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
+
+The AWS bucket contains public and can be accessed without DACO approval.
```
Comment (justincorrigible) [inline docs/data-access/icgc-25k-data.md:29]: "public" what? lol
```suggestion
The AWS bucket contains public data and can be accessed without DACO approval.
```

File: docs/data-access/icgc-25k-data.md [hunk 3]
```diff
@@ -46,6 +49,25 @@ Authentication to the server is done using username and password:
 
 You can connect to this server using any SFTP client of your choice.
 
+### AWS details
+
+The AWS bucket is located at:
```
Comment (justincorrigible) [inline docs/data-access/icgc-25k-data.md:54]: strictly speaking, the url is not it's location, but a pointer to it
```suggestion
The AWS bucket is reachable at: 
```

File: docs/data-access/icgc-25k-data.md [hunk 4]
```diff
@@ -46,6 +49,25 @@ Authentication to the server is done using username and password:
...
+### AWS details
+
+The AWS bucket is located at: 
+- **Host**: `https://object.genomeinformatics.org`
+
+No additional authentication is required.
+
+To navigate and explore the data:
+```
+aws s3 ls s3://icgc25k-open --endpoint-url https://object.genomeinformatics.org --no-sign-request
```
Comment (justincorrigible) [inline docs/data-access/icgc-25k-data.md:61]: naive question: are people expected to know at this point of the instructions that they need an aws client, etc?

Comment (edsu7) [reply docs/data-access/icgc-25k-data.md:61]: We do, as the same is expected for SFTP and filezilla. It was decided at the time, not to endorse.

**Review by justincorrigible — APPROVED** [review]
Squash away :shipit:

File: docs/data-access/icgc-25k-data.md [hunk 5]
```diff
@@ -22,15 +22,18 @@ Files from other contributing projects are all hosted by ICGC's partner reposito
...
 
-A SFTP server is available to access ICGC Release Data and PCAWG data.
+ICGC Release Data and PCAWG data are hosted in two locations: an SFTP server, and an AWS bucket.
 
-The server hosts three data directories with the following data:
+The SFTP contains public and DACO controlled data, only available to authorized DACO-approved users only. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).
+
+The AWS bucket contains public data and can be accessed without DACO approval.
+
+Both locations contain directories with the following data:
```
Comment (joneubank) [inline docs/data-access/icgc-25k-data.md:31]: Let's rearrange this text to state upfront that there are different sources for open and controlled data:

```md
ICGC Release Data contains both open and controlled access data. [Open access release data](#link-to-open-access-header) is stored on a publicly available Object Storage Bucket. [Controlled access release data](#link-to-controlled-access-header) is hosted on an SFTP server that requires ICGC DACO approval to access.
```

File: docs/data-access/icgc-25k-data.md [hunk 6]
```diff
@@ -46,6 +49,25 @@ Authentication to the server is done using username and password:
 
 You can connect to this server using any SFTP client of your choice.
 
+### AWS details
```
Comment (joneubank) [inline docs/data-access/icgc-25k-data.md:52]: Lets rename this to `Open Access Release Data`

File: docs/data-access/icgc-25k-data.md [hunk 7]
```diff
@@ -46,6 +49,25 @@ Authentication to the server is done using username and password:
...
+
+The AWS bucket is reachable at:
```
Comment (joneubank) [inline docs/data-access/icgc-25k-data.md:54]:
```md
Open access release data is hosted on a publicly available Object Storage Bucket. This bucket uses the AWS S3 interface and is therefore accessible using any AWS object storage client.

The bucket is reachable at....
```

**Review by joneubank — COMMENTED** [review]
Rephrased some stuff

**Pull request merged**